### PR TITLE
   Create sign in and recovery password flux technology

### PR DESCRIPTION
**Pull request by airavata-elephant**
1. Create SignInContainer.
2. Create Sign-In-Reducer.
3. Create props in SignIn component.
4. Import SignInReducer in Reduxe.js
5. Replace SinIn component in App.js to SignInConteiner component.

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -11,6 +11,7 @@ import Header from "./component/Header/Header";
 import EditProfile from "./component/RegistrationPages/EditProfile/EditProfile";
 import ChangePassword from "./component/RegistrationPages/ChangePassword/ChangePassword";
 import SignIn from "./component/RegistrationPages/SignIn/SignIn";
```
Comment (Vahtangius) [inline src/App.js:13]: Ctrl+Alt+o

Comment (airavata-elephant) [reply src/App.js:13]: ok thanks

File: src/component/RegistrationPages/SignIn/SignInContainer.jsx [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+let mapDispatchToProps = (dispatch) => {
+  return {
+    updateSignInEmail: (emailText) => {
+      dispatch(changeEmailActionCreator(emailText));
+    },
+    updateSignInPassword: (passwordText) => {
+      dispatch(changePasswordActionCreator(passwordText));
+    },
+
```
Comment (Vahtangius) [inline src/component/RegistrationPages/SignIn/SignInContainer.jsx:22]: Delete empty row

Comment (airavata-elephant) [reply src/component/RegistrationPages/SignIn/SignInContainer.jsx:22]: ок

**Review by Vahtangius — APPROVED** [review]

**Review by Vahtangius — APPROVED** [review]

**Review by TarasOrlovskyi — APPROVED** [review]

**Review by TarasOrlovskyi — APPROVED** [review]

**Pull request merged**